homeassistant/components/control4/climate.py

```diff
@@ -271,6 +282,28 @@ def target_temperature_low(self) -> float | None:
             return data.get(CONTROL4_HEAT_SETPOINT)
         return None
 
+    @property
+    def fan_mode(self) -> str | None:
+        """Return the current fan mode."""
+        data = self._thermostat_data
+        if data is None:
+            return None
+        return data.get(CONTROL4_FAN_MODE)
+
+    @property
+    def fan_modes(self) -> list[str] | None:
+        """Return the list of available fan modes."""
+        data = self._thermostat_data
+        if data is None:
+            return None
+        modes = data.get(CONTROL4_FAN_MODES_LIST)
+        if modes is None:
+            return None
+        # Handle both string (comma-separated) and list formats
+        if isinstance(modes, str):
+            return [m.strip() for m in modes.split(",") if m.strip()]
+        return list(modes) if modes else None
```

Comment:
Okay so how does a control4 system get its fan modes? Are there defaults for that? Can people pick custom ones?

Because ideally if we have fan modes, we make them snake_case and add their translation to strings.json (and icon to icons.json). This way we can show the fan modes localized, but that is only depending on how consistent the source data is.